### DISCUSSION TITLE
how is this different from flutter??? please describe

### DISCUSSION THREAD

**Discussion by fisforfaheem** [slint-ui/slint#440] [Q&A]

**Comment by tronical** [comment]
There are many differences.

For example Flutter is tied to using Dart as programming language. SixtyFPS is not, it currently supports Rust, C++ and NodeJS and there is room for more.

Flutter’s approach to describing the UI is syntactically vastly different from SixtyFPS’s declarative markup language.

What other differences are you looking for, in particular?

(I’m tempted to convert this from a bug tracking issue to a GitHub discussion topic, unless perhaps there is an acceptance criteria that can be put in place for this)

**Comment by Be-ing** [comment]
I'm sure many people wonder how this compares to alternatives like Flutter and Qt. It may be a good idea to add a page to the website explicitly comparing SixtyFPS to these.

**Comment by tronical** [comment]
> I'm sure many people wonder how this compares to alternatives like Flutter and Qt. It may be a good idea to add a page to the website explicitly comparing SixtyFPS to these.

I agree, it’s good to document this. I’d still start with GH discussions though. What do you think?

**Comment by fisforfaheem** [comment]
Yes Please do this... But you cant really build apps like u do in flutter
with this write??



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
08/28/21,
09:32:41 PM

**Comment by Be-ing** [comment]
Comparisons to GTK and Xamarin may be a good idea too.
